draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -226,12 +226,12 @@ not have any effect on connectivity or connection setup time.
 Examples
 ========
 
-The examples below show how the mDNS technique is used during ICE
-processing. The first example shows the typical case, and the 
-other two examples demonstrate how peer-reflexive candidates for
-local IP addresses can be created due to timing differences.
+The examples below show how the mDNS technique is used during ICE processing.
+The first example shows the typical case, and other two examples demonstrate
```

Comment:
Maybe just me, but it reads better with 'the' than without.

Reply:
agreed, seems like this was accidental

Reply:
Yeah, I deleted the word by accident.
```suggestion
The first example shows the typical case, and the other two examples demonstrate
```